pwiz_tools/Skyline/TestPerf/SmallMolLibrariesTutorialTest.cs

```diff
@@ -0,0 +1,431 @@
+                //   �	Enter �20,000� in the Resolving power field.
+                transitionSettingsUI.ProductRes = 20000;
+                //   �	Check Use high-selectivity extraction.
+                transitionSettingsUI.UseSelectiveExtraction = true;
+            });
+
+            //   The Transition Settings form should look like this:
+            PauseForScreenShot("Transition Settings - Full-Scan", 5);
+            //   �	Click the OK button.
+            OkDialog(transitionSettingsUI, transitionSettingsUI.OkDialog);
+            doc = WaitForDocumentChange(doc);
+
+            //   Adding and Exploring a Spectral Library
+            // Before you can explore the library, Skyline must be directed to its location by adding your library of interest to the global list of libraries for document editing.
+            //   To get started with the small molecule library containing Drosophila lipids perform the following steps:
+            //   �	From the Settings menu, click Molecule Settings.
+            //   �	Click the Library tab.
+            var peptideSettingsUI = ShowDialog<PeptideSettingsUI>(() =>
+                SkylineWindow.ShowPeptideSettingsUI(PeptideSettingsUI.TABS.Library));
+            //   �	Click the Edit List button.
+            var libListDlg =
+                ShowDialog<EditListDlg<SettingsListBase<LibrarySpec>, LibrarySpec>>(peptideSettingsUI.EditLibraryList);
+            //   �	Click the Add button in the Edit Libraries form.
+            var addLibDlg = ShowDialog<EditLibraryDlg>(libListDlg.AddItem);
+            var drosophilaLipids = "Drosophila Lipids";
+            RunUI(() =>
+            {
+                //   �	Enter �Drosophila Lipids� in the Name field of the Edit Library form.
+                addLibDlg.LibraryName = drosophilaLipids;
+                //   �	Click the Browse button.
+                //   �	Navigate to the SmallMoleculeLibraries folder created earlier.
+                //   �	Select the �Drosophila_Lipids_Neg.blib� file.
+                addLibDlg.LibraryPath = TestFilesDirs[0].GetTestPath("Drosophila_Lipids_Neg.blib");
+            });
+            //   �	Click the Open button.
+            //   �	Click the OK button in the Edit Library form.
+            OkDialog(addLibDlg, addLibDlg.OkDialog);
+            //   �	Click the OK button in the Edit Libraries form.
+            OkDialog(libListDlg, libListDlg.OkDialog);
+
+            //   The Libraries list in the Molecule Settings form should now contain the Drosophila Lipids library you just created.
+            //   �	Check the Drosophila Lipids checkbox to tell Skyline to use this library in the current document.
+            //   �	If you have any other libraries in this list checked, uncheck them now.
+            RunUI(() => { peptideSettingsUI.PickedLibraries = new[] {drosophilaLipids}; });
+            //   The Molecule Settings form should now look like:
+            PauseForScreenShot("Molecule Settings - Library", 7);
```

Comment:
Use the IForView type in this pause for the Molecule Settings - Library tab

Reply:
right, thanks.